smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -290,6 +291,33 @@ void restTemplateClient() {
                             .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")));
   }
 
+  @Test
+  @org.junit.jupiter.api.Order(1)
```

Comment:
There is already another test with `@Order(1)` and that should be executed at first because it checks the logs at the application start-up: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/144f5ad250c22617688262df04718b1485bf6de9/smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java#L199

So, the new test should not have the `Order(1)`  annotation.